modules/payloads/singles/cmd/unix/pingback_bind.rb

```diff
@@ -37,14 +37,16 @@ def initialize(info = {})
   # Constructs the payload
   #
   def generate
-    return super + command_string
+    parent = super
+    return command_string if parent.nil?
```

Comment:
Guarding against a `nil` super is better as:
```ruby
super.to_s + command_string
```
since `nil.to_s` is the empty string.